packages/flutter/lib/src/cupertino/radio.dart

```diff
@@ -132,24 +132,22 @@ class CupertinoRadio<T> extends StatefulWidget {
   /// The cursor for a mouse pointer when it enters or is hovering over the
   /// widget.
   ///
-  /// If [mouseCursor] is a [WidgetStateMouseCursor],
-  /// [WidgetStateMouseCursor.resolve] is used for the following [WidgetState]s:
+  /// Resolves in the following states:
   ///
   ///  * [WidgetState.selected].
-  ///  * [WidgetState.hovered].
   ///  * [WidgetState.focused].
   ///  * [WidgetState.disabled].
   ///
-  /// If null, then [SystemMouseCursors.basic] is used when this radio button is disabled.
-  /// When this radio button is enabled, [SystemMouseCursors.click] is used on Web, and
-  /// [SystemMouseCursors.basic] is used on other platforms.
+  /// If null, then [SystemMouseCursors.basic] is used when this radio button is
+  /// disabled. When this radio button is enabled, [SystemMouseCursors.click] is
+  /// used on Web, and [SystemMouseCursors.basic] is used on other platforms.
   ///
   /// See also:
   ///
   ///  * [WidgetStateMouseCursor], a [MouseCursor] that implements
   ///    `WidgetStateProperty` which is used in APIs that need to accept
-  ///    either a [MouseCursor] or a [WidgetStateProperty<MouseCursor>].
-  final MouseCursor? mouseCursor;
+  ///    either a [MouseCursor] or a [WidgetStateProperty].
+  final WidgetStateProperty<MouseCursor>? mouseCursor;
```

Comment:
What happens if you explicitly pass `null` here?

Reply:
I think the null coalescing operator should make it resolve to `WidgetStateProperty.all(CupertinoRadio.defaultMouseCursor(widget.onChanged))`.